lib/src/codecs/codec_IMUEvent.cpp

```diff
@@ -86,35 +86,35 @@ std::string IMUevent::getType() const
     return IMUevent::tag;
 }
 
-double IMUevent::convertToSI(int index, int value)
+double IMUevent::convertToSI(int value, int index)
 {
-    static constexpr double ay_bias = 0.12;
     static constexpr double ax_bias = 0.21;
+    static constexpr double ay_bias = 0.12;
     static constexpr double az_bias = 0.06;
-    static constexpr double ay_gain = 9.80665/9.83;
     static constexpr double ax_gain = 9.80665/9.84;
+    static constexpr double ay_gain = 9.80665/9.83;
     static constexpr double az_gain = 9.80665/9.95;
 
 
     switch(index) {
     case(ACC_X):
         return (value * (9.80665 / 16384.0)+ax_bias)*ax_gain;
     case(ACC_Y):
-        return (value * -(9.80665 / 16384.0)+ay_bias)*ay_gain;
+        return -(value * (9.80665 / 16384.0)+ay_bias)*ay_gain;
     case(ACC_Z):
         return (value * (9.80665 / 16384.0)+az_bias)*az_gain;
     case(GYR_X):
+    case(GYR_Y):
```

Comment:
GYRO_Y was in it before:
```
    case(3):
    case(4):
    case(5):
        return val * (250.0 * M_PI / (2.0 * 180.0 * 16384.0));
```
All three (gx, gy, gz) had the same equation. Now GYRO_Y needs to  have its own equation because it is negated. I think this should be reverted.

Reply:
Should I add the same equation from GYR_Z to GYR_X then?

Reply:
just as it was before 
* GYRO_X and GYRO_Z should have the same equation (not negated)
* GYRO_Y needs a different equation because it has the negative sign out the front. 

```
case(GX):
case(GZ):
 return (non negated)
case(GY):
 return (negated)
```

Am I missing something?